docs/release_notes/ibexa_dxp_v5.0.md

```diff
@@ -10,8 +10,135 @@ month_change: false
+
+The default help menu can be modified to include links to internal editorial guidelines, custom tutorials, or support pages.
+
+![Integrated help menu](../administration/back_office/img/5_0_integrated_help_menu.png)
+
+For more information, see [Integrated help](integrated_help.md).
+
+[[= release_note_entry_end() =]]
+
+[[= release_note_entry_begin("Anthropic connector " + version, '2025-12-10', ['Headless', 'Experience', 'Commerce', 'LTS Update', 'New feature']) =]]
+
+This release introduces a new AI connector that you can install and enable to integrate [AI Actions](ai_actions.md) with [Anthropic Claude](https://claude.com/product/overview).
+
+For more information, see [install Anthropic connector](configure_ai_actions.md#install-anthropic-connector).
+
+[[= release_note_entry_end() =]]
+
+[[= release_note_entry_begin("Ibexa DXP " + version, '2025-12-10', ['Headless', 'Experience', 'Commerce', 'New feature']) =]]
+
+#### Security
+
+This release includes security fixes.
+To learn more, see the [corresponding security advisory](https://developers.ibexa.co/security-advisories/ibexa-sa-2025-005-password-change-and-xss-vulnerabilities-in-back-office).
+
+#### Real-time collaborative editing
+
+Real-time editing is now part of the [Collaborative editing](collaborative_editing.md) feature.
+
+By using it, users can edit and review content in real time, making teamwork faster, more efficient, and streamlining the content review process.
+The system automatically tracks changes, allowing seamless collaboration within a single content item.
+
+This extends the already existing capabilities allowing editors to work on the same content created in [[= product_name =]] simultaneously, streamlining the content creation and review process.
+
+![Participants list](img/participants_list.png)
+
+For more information, see [configure Collaborative editing](configure_collaborative_editing.md).
+
+#### Taxonomy suggestions for faster content classification
+
+You can now speed up taxonomy assignment with AI-powered taxonomy suggestions.
+
+Instead of manually browsing through large taxonomy trees and selecting categories or tags one by one, editors can pick from suggestions generated based on the product's or content item's information, for example name and description. 
+
+Such approach helps reduce manual effort, minimize errors, and significantly improve the speed and consistency of content and product classification.
+
+![Taxonomy entries suggested by the AI Assistant](img/taxonomy_suggestions_content.png "Taxonomy entries suggested by the AI Assistant")
+
+For more information, see [Taxonomy suggestions](taxonomy.md#taxonomy-suggestions).
+
+#### Infrastructure
+
+- MariaDB 11.4 is now [officially supported](requirements.md#dbms)
+
+#### Developer experience
+
+##### PHP API
+
+The following additions were made to the PHP API:
+
+##### Real-time collaborative editing:
```

Comment:
I guess this is a part of PHP API additions.
I see two options here:
1. Add one more #.
2. Treat it as a bulleted list.
```suggestion
###### Real-time collaborative editing:
```